src/org/usfirst/frc/team2335/robot/commands/FeedShooter.java

```diff
@@ -0,0 +1,45 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+
+public class FeedShooter extends Command
+{
+    public FeedShooter() 
+    {
+        requires(Robot.shooter);
+    }
+
+    protected void initialize() 
+    {
+    	
+    }
+
+    protected void execute() 
+    {
+        //Turns on feed motor
+    	System.out.println("Running");
```

Comment:
I'll add that if this is debug code, it should optimally be removed from master. Perhaps add it as a code snippet to any codebase-wide documentation, however, as a potential means to help future years' programmers debug the code?